pkg/api/const.go

```diff
@@ -51,38 +51,33 @@ const (
 )
 
 const (
-	// Kubernetes153 is the string constant for Kubernetes 1.5.3
-	Kubernetes153 string = "1.5.3"
-	// Kubernetes157 is the string constant for Kubernetes 1.5.7
-	Kubernetes157 string = "1.5.7"
-	// Kubernetes160 is the string constant for Kubernetes 1.6.0
-	Kubernetes160 string = "1.6.0"
-	// Kubernetes162 is the string constant for Kubernetes 1.6.2
-	Kubernetes162 string = "1.6.2"
-	// Kubernetes166 is the string constant for Kubernetes 1.6.6
-	Kubernetes166 string = "1.6.6"
-	// Kubernetes166 is the string constant for Kubernetes 1.7.0
-	Kubernetes170 string = "1.7.0"
-	// Kubernetes166 is the string constant for Kubernetes 1.7.1
-	Kubernetes171 string = "1.7.1"
-	// KubernetesDefaultVersion is the string constant for current Kubernetes version
-	KubernetesDefaultVersion string = Kubernetes166
+	KubernetesVersionHint17 string = "1.7"
+	KubernetesVersionHint16 string = "1.6"
+	KubernetesVersionHint15 string = "1.5"
+	// Why not 1.7?
```

Comment:
I notice the default version is lagging behind. It is using 1.6.6, shall we use latest?